docs/api/nav.md

```diff
@@ -42,7 +42,11 @@ TODO: Playground Example
 
 ## Using NavLink
 
-TODO: Playground Example
+NavLink is a simplified API when interacting with Nav. Developers can customize the component, pass along component properties, modify the direction of the route animation or define a custom animation when navigating. 
+
+import NavLinkExample from '@site/static/usage/nav/nav-link/index.md';
+
+<NavLinkExample />
```

Comment:
Would it be possible to get rid of the extra space in and above the header in the inline demo?
![ios](https://i.gyazo.com/19dcd49541a1255bba82cef7fd1acc95.png)
![md](https://i.gyazo.com/d3680dfa3d5e74998d6109f58e1ffb0c.png)

Reply:
^ Maybe we just have this demo use the phone shell? Might look a bit nicer